db/seeds.rb

```diff
@@ -75,23 +67,20 @@
     start_date: Time.zone.now.beginning_of_week
   )
   responseobj = student.protocol_subscriptions.first.responses.first
-  responseobj.update_attributes!(
-    open_from: 1.minute.ago,
-    invited_state: Response::SENT_STATE)
-  token = responseobj.initialize_invitation_token!
-  puts "student voormeting: #{Rails.application.routes.url_helpers.root_url}?q=#{student.external_identifier}#{token}"
+  invitation_set = InvitationSet.create!(person: student)
+  responseobj.update_attributes!(open_from: 1.minute.ago, invitation_set: invitation_set)
+  invitation_token = invitation_set.invitation_tokens.create!
+  puts "student voormeting: #{Rails.application.routes.url_helpers.root_url}?q=#{student.external_identifier}#{invitation_token.token_plain}"
   responseobj = student.protocol_subscriptions.first.responses[10]
-  responseobj.update_attributes!(
-    open_from: 1.minute.ago,
-    invited_state: Response::SENT_STATE)
-  token = responseobj.initialize_invitation_token!
-  puts "student dagboek: #{Rails.application.routes.url_helpers.root_url}?q=#{student.external_identifier}#{token}"
+  invitation_set = InvitationSet.create!(person: student)
+  responseobj.update_attributes!(open_from: 1.minute.ago, invitation_set: invitation_set)
+  invitation_token = invitation_set.invitation_tokens.create!
+  puts "student dagboek: #{Rails.application.routes.url_helpers.root_url}?q=#{student.external_identifier}#{invitation_token.token_plain}"
   responseobj = student.protocol_subscriptions.first.responses.last
-  responseobj.update_attributes!(
-    open_from: 1.minute.ago,
-    invited_state: Response::SENT_STATE)
-  token = responseobj.initialize_invitation_token!
-  puts "student nameting: #{Rails.application.routes.url_helpers.root_url}?q=#{student.external_identifier}#{token}"
+  invitation_set = InvitationSet.create!(person: student)
+  responseobj.update_attributes!(open_from: 1.minute.ago, invitation_set: invitation_set)
+  invitation_token = invitation_set.invitation_tokens.create!
+  puts "student nameting: #{Rails.application.routes.url_helpers.root_url}?q=#{student.external_identifier}#{invitation_token.token_plain}"
```

Comment:
Assign these to different persons so the three links don't all go to the same set of questionnaires